frontend/components/groups/js/groups.component.ts

```diff
@@ -143,6 +144,26 @@ class GroupsController implements ng.IController {
 
 	}
 
+	public compareSavedGroup() {
```

Comment:
Not entirely sure what's the purpose of this...

Isn't this essentially overwriting selectedGroup's properties with savedGroup's? is there much point in checking if they're the same?

Reply:
So basically we need a store of the group we're editing so if we go back we can reverse those changes. We could just straight up override each property, if you think that's better?

Reply:
Yea I feel like the comparison is a bit useless. Because they are either identical, or you want to take whatever that's in savedgroup... in other words, you just want savedGroup's data.

Would probably want to rename the function though if that's the case.